packages/react-jss/src/withStyles.js

```diff
@@ -70,8 +62,11 @@ const createWithStyles: CreateWithStyles = <Theme>(styles, options = {}) => {
     })
 
     WithStyles.displayName = `WithStyles(${displayName})`
+
+    // $FlowFixMe[prop-missing]
```

Comment:
not sure if we can but maybe there is a way to avoid this?

Reply:
I could not find